README.md

```diff
@@ -178,20 +161,17 @@ Verdaccio aims to support all features of a standard npm client that make sense
 
 - npm audit - **supported**
 
-<h2 align="center">Contributors</h2>
-
-This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
-<a href="../../graphs/contributors"><img src="https://opencollective.com/verdaccio/contributors.svg?width=890&button=false" /></a>
 
+## Sponsors
 
-### Backers
+#### Open Source License
 
-Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com/verdaccio#backer)]
-
-<a href="https://opencollective.com/verdaccio#backers" target="_blank"><img src="https://opencollective.com/verdaccio/backers.svg?width=890"></a>
+Thanks to the following sponsors to help to achieve our goals providing us free open source licenses.
 
+![jetbrain](assets/sponsor/jetbrains/logo.png)
+![crowdin](assets/sponsor/crowdin/logo.png)
```

Comment:
Should we make these imgs links to their sites?

Reply:
Actually, they don't have a specific size, for that reason I decided to resize images and include them in our assets.

Reply:
good point